reference/paths/apple_pay_certificates__update-apple-pay-certificate.yaml

```
@@ -10,40 +10,13 @@ x-internal: true
 
 requestBody:
   content:
-    application/json:
+    multipart/form-data:
       schema:
-        $ref: "../openapi.yaml#/components/schemas/ApplePayCertificateUpdate"
-      examples:
-        Update Apple Pay certificate record:
-          value:
-            certificate: |-
-              -----BEGIN CERTIFICATE-----\nMIIEbjCCBBSgAwIBAgIIUSayfz2Q+vowC
-              gYIKoZIzj0EAwIwgYAxNDAyBgNVBAMM\nK0FwcGxlIFdvcmxkd2lkZSBEZXZlb
-              G9wZXIgUmVsYXRpb25zIENBIC0gRzIxJjAk\nBgNVBAsMHUFwcGxlIENlcnRpZ
-              mljYXRpb24gQXV0aG9yaXR5MRMwEQYDVQQKDApB\ncHBsZSBJbmMuMQswCQYDV
-              QQGEwJVUzAeFw0yMjEyMTIwOTUyNTVaFw0yNTAxMTAw\nOTUyNTRaMIGfMScwJ
-              QYKCZImiZPyLGQBAQwXbWVyY2hhbnQuY29tLmdyNHZ5LmRl\nbW8xPTA7BgNVB
-              AMMNEFwcGxlIFBheSBQYXltZW50IFByb2Nlc3Npbmc6bWVyY2hh\nbnQuY29tL
-              mdyNHZ5LmRlbW8xEzARBgNVBAsMCk1OSEc0TENYN0cxEzARBgNVBAoM\nCkdyN
-              HZ5LCBJbmMxCzAJBgNVBAYTAlVTMFkwEwYHKoZIzj0CAQYIKoZIzj0DAQcD\nQ
-              gAEk8DpYAFrE/NLRKEzSzTWg+hCL9Jzr5ccJuf2OV9K/cE9WcRHCeoXaStOtbq
-              F\nsC8Gb9uj4lDR3U3cnZdB2upVrKOCAlUwggJRMAwGA1UdEwEB/wQCMAAwHwY
-              DVR0j\nBBgwFoAUhLaEzDqGYnIWWZToGqO9SN863wswRwYIKwYBBQUHAQEEOzA
-              5MDcGCCsG\nAQUFBzABhitodHRwOi8vb2NzcC5hcHBsZS5jb20vb2NzcDA0LWF
-              wcGxld3dkcmNh\nMjAxMIIBHQYDVR0gBIIBFDCCARAwggEMBgkqhkiG92NkBQE
-              wgf4wgcMGCCsGAQUF\nBwICMIG2DIGzUmVsaWFuY2Ugb24gdGhpcyBjZXJ0aWZ
-              pY2F0ZSBieSBhbnkgcGFy\ndHkgYXNzdW1lcyBhY2NlcHRhbmNlIG9mIHRoZSB
-              0aGVuIGFwcGxpY2FibGUgc3Rh\nbmRhcmQgdGVybXMgYW5kIGNvbmRpdGlvbnM
-              gb2YgdXNlLCBjZXJ0aWZpY2F0ZSBw\nb2xpY3kgYW5kIGNlcnRpZmljYXRpb24
-              gcHJhY3RpY2Ugc3RhdGVtZW50cy4wNgYI\nKwYBBQUHAgEWKmh0dHA6Ly93d3c
-              uYXBwbGUuY29tL2NlcnRpZmljYXRlYXV0aG9y\naXR5LzA2BgNVHR8ELzAtMCu
-              gKaAnhiVodHRwOi8vY3JsLmFwcGxlLmNvbS9hcHBs\nZXd3ZHJjYTIuY3JsMB0
-              GA1UdDgQWBBTWCZyJjskSF0jEGjLCOgbZoF69xDAOBgNV\nHQ8BAf8EBAMCAyg
-              wTwYJKoZIhvdjZAYgBEIMQDlDM0YxMTU0QjAwMjMyNDE4NTdF\nMDQzNENENEU
-              xNDg3MkM5MjdDRTAyNkM5NDU3MUUzMzhCODNCOEE5RjkwQTcwCgYI\nKoZIzj0
-              EAwIDSAAwRQIgPapEjAilLTv+a8MwucE1VeJEdnkKtSK8O45RZCpEl9kC\nIQD
-              hz2+moVwlRJstMYB7kArx9k/vK8iLiXsMfiW/lk0T6w==\n-----END CERTIF
-              ICATE-----\n
+        type: object
+        properties:
+          file:
+            type: string
+            format: binary
```

Is it possible to add a description here? Something like
```suggestion
            format: binary
            description: The payment processing certificate file downloaded from the Apple developer console.
```